### DISCUSSION TITLE
Db2 TPC-C benchmark - building the schema

### DISCUSSION THREAD

**Discussion by chimicus** [TPC-Council/HammerDB#76] [Q&A] [answered]
Scenario:

DB2 installation on a test server with no other load
Installed hammerDB to run on the server

Step:
configured the schema with 5 warehouses and 5 virtual users
hit build

What happens:
Schema creation succeeds, all virtual users complete their tasks pretty quickly (about 10 min)
(CPU usage above 65% constantly for the whole run)
Virtual user 1 is stuck at "GATHERING SCHEMA STATISTICS..."
(CPU usage close to 0%, no memory pressure) 
run time is over 2h.

**Comment by abondvt89** [comment]
Is there any update on this issue?

**Comment by memmertoIBM** [comment]
I have not had a chance to attempt to reproduce the issue.

**Comment by chimicus** [comment, marked as answer]
I found out the problem, it is a db2 issue, I was wrapping one of the executables to print some ad-hoc diagnostic and it turns out db2 doesn't c hope well with executable hijacking... This issue can be closed.

Thanks

**Comment by abondvt89** [comment]
@chimicus Thank you for that update.  Glad you got some clarity on the issue.  Closing this issue as requested.
